tests/integration/llm/dataset_prep.py

```diff
@@ -0,0 +1,83 @@
+import urllib.request
+import json
+
+dataset_urls = {
+    "gsm8k":
+    "https://raw.githubusercontent.com/ymwangg/vllm-test/main/dataset/gsm8k.jsonl",
+    "humaneval":
+    "https://raw.githubusercontent.com/ymwangg/vllm-test/main/dataset/humaneval.jsonl",
+    "mbpp":
+    "https://raw.githubusercontent.com/ymwangg/vllm-test/main/dataset/mbpp.jsonl",
+    "mtbench":
+    "https://raw.githubusercontent.com/ymwangg/vllm-test/main/dataset/mt_bench.jsonl"
+}
+
+parameters = {
+    "gsm8k": {
+        "temperature": 0.0,
+        "stop": ["Question:", "</s>", "<|im_end|>"],
+        "max_num_tokens": 256
+    },
+    "humaneval": {
+        "temperature":
+        0.2,
+        "stop": [
+            "\nclass", "\ndef", "\n#", "\n@", "\nprint", "\nif", "\n```",
+            "<file_sep>"
+        ],
+        "max_num_tokens":
+        512
+    },
+    "mbpp": {
+        "temperature":
+        0.1,
+        "stop":
+        ["\nclass", "\nassert", '\n"""', "\nprint", "\nif", "\n<|/", "\n```"],
+        "max_num_tokens":
+        512
+    },
+    "mtbench": {
+        "temperature": 1.0,
+        "top_p": 0.8,
+        "top_k": 40,
+        "max_num_tokens": 512
+    }
+}
+
+
+def gsm8k_parsing(row: dict, parameters):
+    return {"inputs": row["prompt"], "parameters": parameters, "stream": True}
+
+
+def humaneval_parsing(row: dict, parameters):
+    return {"inputs": row["prompt"], "parameters": parameters, "stream": True}
+
+
+def mbpp_parsing(row: dict, parameters):
+    return {"inputs": row["prompt"], "parameters": parameters, "stream": True}
+
+
+def mtbench_parsing(row: dict, parameters):
```

Comment:
We don't need add parameters in dataset any more, we can pass parameters in awscurl command now.

Reply:
customer can choose to override if needed. Those are the good defaults from science